### PR TITLE
Intro.md - Flink/Lite distinction and fixes

### PR DESCRIPTION
**Pull request by jzdr**

### CODE REVIEW

File: docs/scenarios_authoring/BasicNodes.md [hunk 1]
```diff
@@ -108,26 +108,26 @@ Example:
...
 ## UnionMemo
 ![union_memo_window](img/union_memo_window.png)
 
-Similarly to Union, UnionMemo node merges branches into one branch, events are emitted on every incoming event and event time is inherited from the incomming event.
+Similarly to Union, UnionMemo node merges branches into one branch, events are emitted on every incoming event and event time is inherited from the incoming event.
 
 There are however important differences in the way UnionMemo works:
 - events from the incoming branches are matched together based on some key value
-- data that arrived from any of the incoming branches will be memoized by the UnionMemo node for time duration defined in stateTimeout. If new event arrives before stateTimeout, the stateTimeout timer is reset
+- data that arrived from any of the incoming branches will be memorized by the UnionMemo node for time duration defined in stateTimeout. If new event arrives before stateTimeout, the stateTimeout timer is reset
```
Comment (kdutkowski) [inline docs/scenarios_authoring/BasicNodes.md:123]: I think it should stay as 'memoized' https://en.wikipedia.org/wiki/Memoization

File: docs/scenarios_authoring/Intro.md [hunk 2]
```diff
@@ -17,23 +17,23 @@ Nussknacker nodes process events; once the node finishes processing of the event
...
-## Notion of time
+## (Streaming-Flink only) Notion of time
 
 Notion of passing time is very important in dealing with real time events processing. 
 Please see following excellent references to learn about basic concepts:
 * [Notion of time in Flink](https://ci.apache.org/projects/flink/flink-docs-stable/docs/concepts/time/)
 * [Streaming 101: The world beyond batch by O'Reilly](https://www.oreilly.com/radar/the-world-beyond-batch-streaming-101/)
 
-Nussknacker uses Flink, so in general, Flink documentation applies. Certain Nussknacker components make assumptions and have predefined settings, so that the end users don't have to configure all by themselves.
+For Nussknacker's Flink engine Flink documentation applies. Certain Nussknacker components make assumptions and have predefined settings, so that the end users don't have to configure all by themselves.
```
Comment (kdutkowski) [inline docs/scenarios_authoring/Intro.md:27]: How about "For Nussknacker's Flink engine Flink documentation applies." -> "For Streaming-Flink engine Flink documentation applies."

File: docs/scenarios_authoring/BasicNodes.md [hunk 3]
```diff
@@ -79,15 +79,15 @@ Switch distributes incoming records among output branches in accordance with the
...
+Each record from the `source` is tested against the condition defined on the edge. If `#color` is `blue` record goes to the `blue sink`.  If `#color` is `green` record goes to the `green sink`. For every other value record goes to the `sink for others`.
 
 ![switch window](img/switch_window.png)
 
 The Switch node takes two parameters: `Expression` and `exprVal`. `Expression` contains expression which is evaluated for each record; result is assigned to the variable configured in `exprVal` entry field - `#color` in the example above.
  
 ![switch_edge_condition](img/switch_edge_condition.png)
 
-Eeach edge outgoing from `Switch` node has a boolean expression attached to it; if the expression evaluates to true the record is allowed to pass through this edge. Record go to the first output with matching condition. *Order of matching outgoing edges is not guaranteed.*
+Each edge outgoing from `Switch` node has a boolean expression attached to it; if the expression evaluates to true the record is allowed to pass through this edge. Record goes to the first output with matching condition. *Order of matching outgoing edges is not guaranteed.*
```
[the remark this answers is not in the document]
Comment (jzdr) [reply docs/scenarios_authoring/BasicNodes.md:90]: Changed to "Each outgoing edge of Switch"

File: docs/scenarios_authoring/Intro.md (hunk 2 above)
Comment (jzdr) [reply docs/scenarios_authoring/Intro.md:27]: fixed

**Review by mproch — APPROVED** [review]

**Pull request merged**